lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -220,7 +226,8 @@ sub _reset {
 sub _serve {
   my $c        = shift;
   my $checksum = $c->stash('checksum');
-  my $asset    = $c->asset;
+  my $helper = $plugin->config->{helper};
```

Comment:
So this could be `my $helper = $c->stash("asset.helper")` and stored by doing `$app->defaults("asset.helper" => $config->{helper})`